src/bsp/BspMerger.cpp

```diff
@@ -825,8 +828,8 @@ bool BspMerger::merge(Bsp& mapA, Bsp& mapB) {
 
 	BSPPLANE separationPlane = separate(mapA, mapB);
 	if (separationPlane.nType == -1) {
-		logf("No separating axis found. The maps overlap and can't be merged.\n");
-		return false;
+		logf("Warning! No separating axis found. The maps be overlap.\n");
+		//return false;

```

Comment:
But i want to merge map with another's models, I can do it?(or how to create empty worldspawn to avoid this)

Reply:
If you want the models to overlap, then the other map's models need to be submodels and used with entities instead of being merged into the worldspawn model. Otherwise you'd need to re-implement all of the compile tools to get proper BSP generation working.

Reply:
@wootguy  I tried it in new commit. But Unsuccessfully .... :(